tests/Datagrid/DatagridTest.php

```diff
@@ -78,8 +78,10 @@ protected function setUp(): void
 
         $this->formBuilder
             ->method('add')
-            ->willReturnCallback(function ($name, $type, $options) use ($eventDispatcher, $formFactory): void {
+            ->willReturnCallback(function ($name, $type, $options) use ($eventDispatcher, $formFactory) {
```

Comment:
```suggestion
            ->willReturnCallback(function ($name, $type, $options) use ($eventDispatcher, $formFactory): FormBuilderInterface {
```
We must import this interface.

Reply:
Done.